ports/nghttp2/portfile.cmake

```
@@ -13,13 +13,27 @@ vcpkg_from_github(
     HEAD_REF master
 )
 
```

```suggestion

if (VCPKG_LIBRARY_LINKAGE STREQUAL static)
    set(ENABLE_STATIC_LIB ON)
    set(ENABLE_SHARED_LIB OFF)
else()
    set(ENABLE_STATIC_LIB OFF)
    set(ENABLE_SHARED_LIB ON)
endif()
```
We just need to add these codes here.

Then add the following codes to OPTIONS in `vcpkg_configure_cmake()`.
```
-DENABLE_STATIC_LIB=${ENABLE_STATIC_LIB}
-DENABLE_SHARED_LIB=${ENABLE_SHARED_LIB}
```

---

done